### ISSUE TITLE
`load_recipes': undefined method `each' for String (NoMethodError)

### ISSUE THREAD

**Issue by neidiom** [itamae-kitchen/itamae#345]
I am getting this error. Is it possible to return a more user-friendly error like "Array expected"

````
ruby it.rb 
 INFO : Starting Itamae... 
DEBUG : Executing `mkdir -p /tmp/itamae_tmp`...
DEBUG :   exited with 0
DEBUG : Executing `chmod 777 /tmp/itamae_tmp`...
DEBUG :   exited with 0
/home/nedim/.rvm/gems/ruby-3.0.3/gems/itamae-1.12.5/lib/itamae/runner.rb:42:in `load_recipes': undefined method `each' for "/home/nedim/neidiom/rail7_test/lib/itamae/nginx.rb":String (NoMethodError)
	from /home/nedim/.rvm/gems/ruby-3.0.3/gems/itamae-1.12.5/lib/itamae/runner.rb:12:in `run'
	from it.rb:18:in `<main>'
````

It expects an array, not a string that is clear.

This is my code that fails

````
require 'itamae'
role_file = File.expand_path('../lib/itamae/nginx.rb', __FILE__)
backend_type = 'local'
options = {}
options[:invoked_from_cli] = false
options[:log_level] = 'info' unless options[:log_level]
options[:color] = true unless options[:color]
options[:host] = 'local'
options[:type] = 'local'
Itamae::Runner.run(role_file, backend_type, options)
````

But if I use it like this, there is no problem

````
require 'itamae'

role_file = File.expand_path('../lib/itamae/nginx.rb', __FILE__)

recipe_files = []
recipe_files.append(role_file)


backend_type = 'local'

options = {}
options[:invoked_from_cli] = false
options[:log_level] = 'info' unless options[:log_level]
options[:color] = true unless options[:color]
options[:host] = 'local'
options[:type] = 'local'
Itamae::Runner.run(recipe_files, backend_type, options)
````

**Comment by unasuke** [comment]
> user-friendly error like "Array expected"

@neidiom That sounds good! Could you create a pull request?

**Comment by dvinciguerra** [comment]
Can I do this pull request? 😊

**Comment by neidiom** [comment]
please do @dvinciguerra

**Comment by unasuke** [comment]
I published v1.14.1 Thank you @neidiom and @dvinciguerra !

* https://github.com/itamae-kitchen/itamae/blob/master/CHANGELOG.md#v1141
* https://rubygems.org/gems/itamae/versions/1.14.1
